pkg/vcs/linux.go

```diff
@@ -8,19 +8,26 @@ import (
 	"io"
 	"net/mail"
 	"path/filepath"
+	"regexp"
 	"sort"
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/google/syzkaller/pkg/email"
 	"github.com/google/syzkaller/pkg/osutil"
 )
 
 type linux struct {
 	*git
 }
 
+type RecipientInfo struct {
```

Comment:
Since this is used in vcs.go in Commit struct in OS-independent code, this should go into vcs.go after the Commit declaration. All OSes should now use this, this is not Linux-specific.